### DISCUSSION TITLE
Self Modifying or Replacing Tags

### DISCUSSION THREAD

**Discussion by TheMouster** [defx/synergy#12] [Q&A]
Hi,

Is it possible with Synergy to create a tag that modifies itself or replaces itself e.g. <map-link lat=999.99 lng=999.99></map-link> transforming into <map-link onclick=‘etc.’ class=‘etc.’></map-link> or <i onclick=‘etc.’ class=‘etc.’></i>?

Thanks

**Comment by defx** [comment]
Hi @TheMouster, apologies for the late reply - I've been on hiatus for the last few months. I'm not sure I understand the question exactly but if you can provide an example then I'm sure I'll be able to help point you in the right direction!

**Comment by TheMouster** [comment]
Hi deft,

Goodness knows what I was thinking at the time I wrote that, or why I think I needed it. Thanks for the answer. Sorry about the scrambled question.

Cheers
